### ISSUE TITLE
Getting started entries in Launcher (v1.5.3) containing wrong links

### ISSUE THREAD

**Issue by OdinsAuge** [SiliconStudio/xenko#465]
The "Getting started" entries open a link that is not valid, e.g.:
"Create and setup new game" --> http://doc.xenko.com/1.8/manual/getting-started/howto-create-and-setup-new-game.html
This entry does not exist in the documentation.
Another example:
"Scripting" --> http://doc.xenko.com/1.8/manual/getting-started/howto-use-scripts.html
http://doc.xenko.com/1.8/manual/get-started/introduction-to-scripting.html is the correct link.

**Comment by xen2** [comment]
Good point, we forgot to update https://github.com/SiliconStudio/xenko-docs/blob/master-1.8/studio_getting_started_links.txt
